app/src/main/java/ar/com/wolox/android/cookbook/room/RoomRecipeActivity.kt

```diff
@@ -0,0 +1,28 @@
+package ar.com.wolox.android.cookbook.room
+
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.wolmo.core.activity.WolmoActivity
+
+/**
+ * To implement DB example:
+ * 1. Add dependencies (See "Notes" at bottom)
```

Comment:
The notes aren't necessary since the Room dependencies are very clear on the gradle.